Source/Core/Core/PowerPC/Jit64/Jit.h

```diff
@@ -56,7 +58,8 @@ class Jit64 : public JitBase, public QuantizedMemoryRoutines
   // Jit!
 
   void Jit(u32 em_address) override;
-  u8* DoJit(u32 em_address, JitBlock* b, u32 nextPC);
+  bool SetEmitterState();
```

Comment:
This should likely have a comment describing what this actually does, since the name is quite vague.